README.md

```diff
@@ -26,12 +26,12 @@ npm start
 ```
 5. Start the backend API.
 
-6.Tubular page will be available at http://localhost:3000
+6. Tubular page will be available at http://localhost:3000
 
 ## Terminating the Application
 The app can be stopped from the terminal emulator by pressing Control-C in their tabs.
```

Comment:
"The app" doesn't match "their tabs".

"its tabs" if you're just talking about this app.